tests/AutoReview/ProjectCodeTest.php

```diff
@@ -339,6 +339,7 @@ private function getTestClasses()
             ->in(__DIR__.'/..')
             ->exclude(array(
                 'Fixtures',
+                'Test',
```

Comment:
hm... I'm not sure do we want to hide it, some devs relys on them, and they are written in very generic form.